tethys_portal/routing.py

```diff
@@ -1,22 +1,46 @@
+from django.conf.urls import url as django_url
+
 from channels.auth import AuthMiddlewareStack
 from channels.routing import ProtocolTypeRouter, URLRouter
+from channels.http import AsgiHandler
 
 from tethys_apps.harvester import SingletonHarvester
 
 harvester = SingletonHarvester()
 app_ws_patterns = harvester.get_url_patterns()['ws_url_patterns']
 
+app_handler_patterns = harvester.get_handler_patterns()
+app_http_handler_patterns = app_handler_patterns['http_handler_patterns']
+app_ws_handler_patterns = app_handler_patterns['ws_handler_patterns']
+
 ws_routing_patterns = []
 
 for namespace, urls in app_ws_patterns.items():
     for url in urls:
         ws_routing_patterns.append(url)
 
+http_routing_patterns = []
+
+for namespace, urls in app_http_handler_patterns.items():
+    for url in urls:
+        http_routing_patterns.append(url)
+
+for namespace, urls in app_ws_handler_patterns.items():
+    for url in urls:
+        ws_routing_patterns.append(url)
+
 application = ProtocolTypeRouter({
-    # Empty for now (http->django views is added by default)
```

Comment:
Remove the "Empty for now" and parentheses portion of this comment. I think it is a good reminder to note that the view urls are added automatically.